src/lib.rs

```diff
@@ -3,18 +3,24 @@
 
 #[doc(no_inline)]
 pub use arbitrary_int;
-pub use bilge_impl::{bitsize, bitsize_internal, DebugBits, FromBits, TryFromBits, BinaryBits, DefaultBits};
+pub use bilge_impl::{bitsize, bitsize_internal, BinaryBits, DebugBits, DefaultBits, FromBits, TryFromBits};
 
 /// used for `use bilge::prelude::*;`
 pub mod prelude {
     #[doc(no_inline)]
     pub use super::{
-        bitsize, bitsize_internal,
-        Bitsized,
-        DebugBits, FromBits, TryFromBits, BinaryBits, DefaultBits,
         // we control the version, so this should not be a problem
         arbitrary_int::*,
-        Filled, assume_filled
+        assume_filled,
+        bitsize,
+        bitsize_internal,
+        BinaryBits,
+        Bitsized,
+        DebugBits,
+        DefaultBits,
+        Filled,
+        FromBits,
+        TryFromBits,
```

Comment:
^this is kinda dumb

Reply:
```rust
#[doc(no_inline)]
pub use super::{bitsize, bitsize_internal, Bitsized};
pub use super::{DebugBits, FromBits, TryFromBits, BinaryBits};
pub use super::arbitrary_int::*;
pub use super::{Filled, assume_filled};
```
this seems reasonable (except rustfmt decides to be jerk and reorders them, oh well)

Reply:
Though they all need no_inline I think. Actually I haven't looked at doc output in a while, we should do that.